### PR TITLE
Update office-365-ti.md

### LINKED ISSUE #12520: O365 TI Connection Licensing clarification
the text states that we need to have o365 E5, what about M365 E5/G5, will these provide the necessary functionality? if so please clarify. 

What role is required to enable this feature? Please add this to the table

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d4352228-5f34-5590-9721-a1d7cab2ac38
* Version Independent ID: 2a484437-5f3c-ab9f-b10e-67cd71f111a4
* Content: [Threat investigation & response capabilities in Microsoft Defender for Office 365](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/office-365-ti?view=o365-worldwide&redirectSourcePath=%252fen-us%252farticle%252fOffice-365-Threat-Intelligence-overview-32405DA5-BEE1-4A4B-82E5-8399DF94C512)
* Content Source: [microsoft-365/security/office-365-security/office-365-ti.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/office-365-ti.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @MSFTTracyP
* Microsoft Alias: **tracyp**

### PR DESCRIPTION
**Pull request by msbemba**
Added All e5 licenses and an important section to the document

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12520

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/office-365-ti.md [hunk 1]
```diff
@@ -57,7 +57,11 @@ To view and use this report in the Microsoft 365 Defender portal at <https://sec
 
 #### Office 365 Threat Intelligence connection
 
-This feature is only available if you have an active Office 365 E5 subscription or the Threat Intelligence add-on. For more information, see the Office 365 Enterprise E5 product page.
+> [!IMPORTANT]
+> This setting was used when Microsoft Defender for Office 365 and Microsoft Defender for Endpoint were in different portals previously. After the convergence of security experiences into a unified portal that is now called Microsoft 365 Defender, these settings are irrelevant and don't have any functionality associated with them. You can safely ignore the status of the control until it is removed from the portal.
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/office-365-ti.md:62]:
```suggestion
> This setting was used when Microsoft Defender for Office 365 and Microsoft Defender for Endpoint were in different portals previously. After the convergence of security experiences into a unified portal that is now called Microsoft 365 Defender, these settings are irrelevant and don't have any functionality associated with them. You can safely ignore the status of the control until it is removed from the portal.
```

Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/office-365-ti.md:62]:
```suggestion
```

File: microsoft-365/security/office-365-security/office-365-ti.md [hunk 2]
```diff
@@ -57,7 +57,11 @@ To view and use this report in the Microsoft 365 Defender portal at <https://sec
...
+
+This feature is only available if you have an active Office 365 E5/G5, M365 E5/G5 subscription or the Threat Intelligence add-on. For more information, see the Office 365 Enterprise E5 product page.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/office-365-ti.md]:
```suggestion
This feature is only available if you have an active Office 365 E5 or G5 or Microsoft 365 E5 or G5 subscription or the Threat Intelligence add-on. For more information, see the Office 365 Enterprise E5 product page.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@MSFTTracyP Gentle reminder

**Pull request merged**